src/components/Comment/index.tsx

```diff
@@ -159,26 +200,7 @@ export default class CommentComponent extends React.Component<CommentProps> {
     const newReplyHidden = hideNewReply || replyBeingEdited;
 
     let replyActions = <></>;
-    if (!newReplyHidden && isFocused && comment.newReply.length > 0) {
-      replyActions = (
-        <div className="comment__reply-actions">
-          <button
-            type="submit"
-            className="comment__button comment__button--primary"
-          >
-            {strings.REPLY}
-          </button>
-          <button
-            type="button"
-            onClick={onClickCancelReply}
-            className="comment__button"
-          >
-            {strings.CANCEL}
-          </button>
-        </div>
-      );
-    }
-
+    let replyForm = <></>;
     let replyTextarea = <></>;
```

Comment:
Why not just `null`?

```suggestion
    let replyForm = null;
    let replyTextarea = null;
```